text/0000-colocated-tests.md

```diff
@@ -0,0 +1,161 @@
+- Start Date: 2020-03-02
+- Relevant Team(s): Ember CLI
+- RFC PR: [#599](https://github.com/emberjs/rfcs/pull/599)
+- Tracking: (leave this empty)
+
+# Co-located Tests
+
+## Summary
+
+As apps grow, and need to be refactored, the ability to have tests be co-located with their concerned tested behavior enables faster refactors and better overall upkeep as related files are grouped together.
```

Comment:
Disagree that co-location makes development an easier so this statement will need some evidence to back it up. I very much prefer tests to be in their own space and not cluttering up the already extremely crowded components directory. The ember language server in VSCode now inserts a link to the test at the top of the file anyway making it very easy to navigate to.

Reply:
good point -- I should bring in some of the philosophy around module unification in here. 
where, drag-and-drop refactoring is allowed, because all folder sub-tree are totally valid, and should remain valid as you move them around your app.

Reply:
I, and many others, don't use VS Code, so.. while tooling can solve part of the problem, it's not all of the problem! <3

Reply:
I've updated the Summary and Motivation sections as well as added tooling as an alternative (in the alternatives section)